### DISCUSSION TITLE
Projects Widget

### DISCUSSION THREAD

**Discussion by charlieforward9** [community/community#140799] [Mobile]
<details>
<summary>Display a project view on the iOS widget to provide a more comprehensive view of current work/iterations</summary>
GitHub for iOS, Version 1.179.0 (142164528)
Device: iPhone 15 (iOS 18.0)
</details>

**Comment by stevepopovich** [comment]
Hi @charlieforward9 I have passed this request onto the team for consideration. Thanks for writing in!
